src/test/app/NFToken_test.cpp

```diff
@@ -7417,6 +7416,164 @@ class NFTokenBaseUtil_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testNFTIssuerIsIOUIssuer(FeatureBitset features)
```

Comment:
@scottschurr  isn't this test redundant with the tests you introduced in fixEnforceNFTokenTrustline?

Reply:
Yes, they are.  We'll have to see which amendment gets into develop first.  If this one goes in first (which seems likely) then I'll need to remove the nearly identical tests from the `fixEnforceNFTokenTrustline` pull request.

Good spotting.